08-applying-material-design-to-compose/projects/final/app/src/main/java/com/yourcompany/android/jetnotes/theme/Theme.kt

```diff
@@ -47,7 +47,7 @@ import androidx.compose.ui.graphics.Color
 private val LightThemeColors = lightColors(
   primary = rwGreen,
   primaryVariant = rwGreenDark,
-  secondary = rwGreen
+  secondary = rwRed
```

Comment:
@denis-buketa shouldn't we update this values removing the `rw` prefix to align them with the changes in the text here? https://github.com/kodecocodes/jet/commit/c817a6def857488de58bc39ddc2547e1cfc8ece7#diff-5ebfd57b0a4da4fb08ab620310e244b9ac7f5411702e1d7fcd01e0e96c89390fR1544